pages/common/httping.md

```diff
@@ -2,9 +2,9 @@
 
 > Measure the latency and throughput of a web server.
 
-- Ping the web server on `host`:
+- Ping the `url`, for example: `httping -g http://github.com:80/tldr-pages/tldr`:
```

Comment:
The backticks aren't required for `url`. 🙂 And it's probably not necessary to add an example in the description.

Perhaps something like this?
```suggestion
- Ping the specified url:
```

But `web server` keeps it consistent with the other examples.

Reply:
all right, let's take this : )